docker-storage-setup.sh

```diff
@@ -878,34 +874,35 @@ get_existing_storage_driver() {
   return 1
 }
 
-docker_root_volume_exists() {
-  lvs $VG/$DOCKER_ROOT_LV_NAME > /dev/null 2>&1 && return 0
+root_volume_exists() {
+  lvs $VG/$ROOT_LV_NAME > /dev/null 2>&1 && return 0
   return 1
 }
 
-# This returns the mountpoint of $DOCKER_ROOT_LV_NAME
-docker_root_lv_mountpoint() {
+# This returns the mountpoint of $ROOT_LV_NAME
+root_lv_mountpoint() {
   local mounts
-  mounts=$(findmnt -n -o TARGET --first-only --source /dev/$VG/$DOCKER_ROOT_LV_NAME)
+  mounts=$(findmnt -n -o TARGET --first-only --source /dev/$VG/$ROOT_LV_NAME)
   echo $mounts
 }
 
-# Mount docker root volume on $DOCKER_ROOT_DIR.
+# Mount container runtime root volume on $ROOT_VOLUME_MOUNT_DIR.
 # drop a systemd mount unit configuration file at /etc/systemd/system.
-# This would allow the $DOCKER_ROOT_DIR mountpoint to persist across reboots.
-mount_docker_root_volume() {
+# This would allow the $ROOT_VOLUME_MOUNT_DIR mountpoint to persist across reboots.
+mount_root_volume() {
   local filename
-  # filename must match $DOCKER_ROOT_DIR path. e.g if $DOCKER_ROOT_DIR is /var/lib/docker
-  # then filename will be var-lib-docker.mount
-  filename=$(systemd_escaped_filename ${DOCKER_ROOT_DIR})
+  # filename must match $ROOT_VOLUME_MOUNT_DIR path. 
+  # e.g if $ROOT_VOLUME_MOUNT_DIR is /var/lib/containers
+  # then filename will be var-lib-containers.mount
+  filename=$(systemd_escaped_filename ${ROOT_VOLUME_MOUNT_DIR})
   cat <<EOF > /etc/systemd/system/$filename
 [Unit]
-Description=Mount docker-root-lv on docker root directory.
+Description=Mount $ROOT_LV_NAME on container runtime root directory.
```

Comment:
Again, I think it  is better to get rid of words `container runtime`. It is enough to say `Mount X on Y directory`